.prettierignore

```diff
@@ -31,3 +31,7 @@ public/assets
 storage
 tmp
 vendor
+terraform/custom_domains/environment_domains/workspace_variables/afqts_dev.tfvars.json
+terraform/custom_domains/environment_domains/workspace_variables/afqts_preprod.tfvars.json
+terraform/custom_domains/environment_domains/workspace_variables/afqts_production.tfvars.json
+terraform/custom_domains/environment_domains/workspace_variables/afqts_test.tfvars.json
```

Comment:
We shouldn't be ignoring these files, can we ensure they pass the linter instead?

Reply:
Files have been successfully linted against Prettier and updated.